src/summary/history/patient-notes/notes-card.component.tsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import { match } from "react-router";
+
+import dayjs from "dayjs";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+
+import { getEncounters } from "./encounter.resource";
+import SummaryCard from "../../cards/summary-card.component";
+import style from "./notes-card-style.css";
+import { Link } from "react-router-dom";
+
+export default function NotesCard(props: NotesCardProps) {
```

Comment:
I do not like the idea of sharing the same component for LevelOne and LevelTwo routes. They are too different. For example, LevelTwo allows for sorting and pagination whereas LevelOne does not. Other differences include that the headers are capitalized in LevelTwo but not in LevelOne, and also that there is a Show More button in LevelOne and not in LevelTwo.

In my opinion, combining the two makes things significantly harder. Instead, we could create smaller components for reuse. Something like this:

```js
<PatientChartTable>
  <PatientChartHeader>
    <tr>Date</tr>
    <tr>Encounter Type, </tr>
  </PatientChartHeader>
  <PatientChartTableRow>
  </PatientChartTableRow>
</PatientChartTable>
```

You might need all of these components. But my point is that we're making abstractions in the wrong place right now and what you've built won't benefit other LevelTwoRoutes that are similar, such as Height + Weight or Vitals.

The component here has